src/algebra/order/archimedean.lean

```diff
@@ -276,6 +276,30 @@ archimedean_iff_nat_lt.trans
  λ H x, let ⟨n, h⟩ := H x in ⟨n+1,
    lt_of_le_of_lt h (nat.cast_lt.2 (lt_add_one _))⟩⟩
 
+lemma archimedean_iff_int_lt : archimedean α ↔ ∀ x : α, ∃ n : ℤ, x < n :=
+begin
+  rw archimedean_iff_nat_lt,
+  refine ⟨λ h x, _, λ h x, _⟩,
+  { rcases h x with ⟨n, h⟩,
+    refine ⟨n, _⟩,
+    exact_mod_cast h },
+  { rcases h x with ⟨n, h⟩,
+    refine ⟨n.to_nat, h.trans_le _⟩,
+    exact_mod_cast int.le_to_nat _ }
+end
+
+lemma archimedean_iff_int_le : archimedean α ↔ ∀ x : α, ∃ n : ℤ, x ≤ n :=
```

Comment:
I think a similar remark to the one I made at `archimedean_iff_int_lt` applies here if we add the missing lemma `exists_int_ge`.

Reply:
I've now copied essentially the same proof as used for `archimedean_iff_nat_le` and `archimedean_iff_rat_le` (which doesn't need that new lemma).